src/_posts/2020-09-17-cassandra-changelog-2

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title: "Apache Cassandra Changelog #2 | December 2020"
+date:   2020-12-01 x
+author: the Apache Cassandra Community
+categories: blog
+---
+![Apache Cassandra Changelog Header](img/changelog_header.jpg "image_tooltip")
+
+Our monthly roundup of key activities and knowledge to keep the community informed.
+
+## Release Notes
+
+#### **Released**
+Apache #Cassandra 4.0-beta3, 3.11.9, 3.0.23, and 2.2.19 were released on November 4 and are [in the repositories](https://cassandra.apache.org/download/). Please pay attention to release notes and let the community know if you encounter problems. Join the Cassandra [mailing list](https://cassandra.apache.org/community/) to stay updated.
+
+#### **Changed**
+Cassandra 4.0 is progressing toward GA. There are 1,390 total tickets and remaining tickets represent 5.5% of total scope. Read the [full summary](https://lists.apache.org/thread.html/r9c6cc19f67d3259e64abbe2b960b8914476d9bfa2463d57c6d5cc44b%40%3Cdev.cassandra.apache.org%3E) shared to the dev mailing list and take a look at the [open tickets](https://issues.apache.org/jira/secure/RapidBoard.jspa?rapidView=355&quickFilter=1661&quickFilter=1658) that need reviewers.
+
+Cassandra 4.0 will be dropping support for older distributions of CentOS 5, Debian 4, and Ubuntu 7.10. [Learn more](https://lists.apache.org/thread.html/r513c507ef19a8524ad5389e548f3d5bbfeb9e7747f3ae4c57ec27329%40%3Cdev.cassandra.apache.org%3E).
+
+## Community Notes
+
+_Updates on Cassandra Enhancement Proposals (CEPs), how to contribute, and other community activities._
+
+#### **Added**
+The community weighed options to address reads inconsistencies for Compact Storage as noted in ticket [CASSANDRA-16217](https://issues.apache.org/jira/browse/CASSANDRA-16217) (committed). The conversation continues in ticket [CASSANDRA-16226](https://issues.apache.org/jira/browse/CASSANDRA-16226) with the aim of ensuring there are no huge performance regressions for common queries when you upgrade from 2.x to 3.0 with Compact Storage tables or drop it from a table on 3.0+.
+
+#### **Added**
+In 4.0 there is a plan to drop compact storage and related code, but the community noticed behavioral changes such as differences in visible behavior between "normal" and compact tables. Solutions are being discussed as part of [CASSANDRA-16217](https://issues.apache.org/jira/browse/CASSANDRA-16217). 
```

Comment:
I think this item and the one above (line 27) are the same thing…?

Reply:
Ah yes, thank you! One should have been deleted - whichever one sounds better please delete the other.

Reply:
@michaelsembwever Shall I remove and resubmit?